content/blog/platypus-platform-pulumi-at-1000-nodes/index.md

```diff
@@ -0,0 +1,119 @@
+
+### Auditability
+
+Every action a user takes in Pulumi can be tracked via the [audit log](https://www.pulumi.com/docs/pulumi-cloud/audit-logs/) which is searchable in two clicks from the Pulumi Cloud homepage dashboard. Audit logs can be filtered by user with one more click. Creating automated backups of your audit logs is a [first-class feature](https://www.pulumi.com/docs/pulumi-cloud/audit-logs/#automated-export). You will never have to worry about responding quickly when someone asks about an event that happened in your system. Also, each deployment and update has logs directly visible from the Pulumi Cloud app, regardless of how it was initiated.
+
+{{< figure src="/images/docs/guides/self-hosted/auditlogs.png" caption="Figure: Viewing the audit log in Pulumi Cloud">}}
+
+### Developer Experience
+
+Probably the most compelling aspect of Pulumi is the developer experience. [Developers love Pulumi](https://www.pulumi.com/testimonials/), because they get to use their preferred tools. General purpose programming languages, visual IDEs, command-line tools, and products with an API-driven architecture are what developers want, and it's what Pulumi delivers in spades.
+
+With Pulumi templates and custom internal component resources in place, developers can drive their own DevOps, provisioning their own infrastructure resources and managing their own deployments directly, reducing bottlenecks in platform teams. Product engineering teams can self-service with a stream-lined workflow that stays compliant with company policy by default. Deep in the code of their favorite programming languages, your developers will never even know they are following the company rules.
+
+{{< figure src="pulumi-ide.png" caption="Figure: Using C# to write a Pulumi program in VS Code">}}
+
+### More to Come
+
+So now that we’ve made a case for how Pulumi can be applied to meet the most pressing needs of a larger organization, hopefully you will realize that the Platypus Platform we will be presenting is more than just infrastructure-as-code. Pulumi is a platform for teams, where your developer portal is not just a catalog of software, but a fully functional control-plane across all your cloud environments.
```

Comment:
well technically Pulumi is a platform to build a platform for teams right?

Reply:
Yep. Changed to "Platypus is a Pulumi-powered platform for teams..."